### ISSUE TITLE
[com.google.fonts/check/layout_valid_language_tags] RHG language tags causes a FAIL 

### ISSUE THREAD

**Issue by khaledhosny** [fonttools/fontbakery#3700]
## Observed behaviour

A font with `RHG` language tag in a FAIL in `com.google.fonts/check/layout_valid_language_tags` even though the tag is valid according to OpenType language tag registry: https://docs.microsoft.com/en-us/typography/opentype/spec/languagetags

## Expected behaviour

Font passes.

## Resources and exact process needed to replicate

[Scheherazade New](https://software.sil.org/downloads/r/scheherazade/ScheherazadeNew-3.300.zip)

```
$ fontbakery check-profile fontbakery.profiles.layout ScheherazadeNew-Regular.ttf -C
Start ... running 3 individual check executions.
 >> com.google.fonts/check/layout_valid_language_tags
    Does the font have any invalid language tags?
    with /Users/khaled/Downloads/ScheherazadeNew-3.300/ScheherazadeNew-Regular.ttf

      Rationale:
      Incorrect language tags can be indications of typos, leftover debugging
      code or questionable approaches, or user error in the font editor. Such
      typos can cause features and language support to fail to work as intended.

     FAIL The following invalid language tags were found in the font: RHG
          [code: bad-language-tags]


    Result: FAIL


Total:

    ERROR: 0
    FAIL: 1
    WARN: 0
    INFO: 0
    SKIP: 0
    PASS: 2

[PPF] 100%

DONE!

    Meaning of check results:

    An ERROR is something wrong with FontBakery itself, possibly a bug.
    A FAIL is a problem with the font that must be fixed.
    A WARN is something that you should consider addressing.
    An INFO result simply prints something useful. Typically stats.
    A PASS means the font looks good for the given checking routine.
    And a SKIP happens when the check does not apply to the given font.

    If you get ERRORs, please help us improve the tool by reporting them at
    https://github.com/googlefonts/fontbakery/issues

    (but other kinds of bug reports and/or
     feature requests are also always welcome, of course!)
```

**Comment by bobh0303** [comment]
Thanks Khaled. This was an issue I should I have created months ago. 

As it turns out, there are quite a number of langtags that have been recently added to the OpenType spec in recent months, RHG among them. IMO, FontBakery needs a process to regularly import registered tags (features, scripts, and languages) from the OpenType specification.

**Comment by bobh0303** [comment]
This would be fixed by simoncozens/opentypespec-py#2

**Comment by simoncozens** [comment]
This should now work.
